src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -90,6 +90,24 @@ namespace AppInstaller::CLI::Workflow
         {
             info << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelReleaseNotesUrl << ' ' << releaseNotesUrl << std::endl;
         }
+        auto documentations = manifest.CurrentLocalization.Get<Manifest::Localization::Documentations>();
+        if (!documentations.empty())
+        {
+            context.Reporter.Info() << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelDocumentations << std::endl;
+            for (const auto& documentation : documentations)
+            {
+                if (!documentation.DocumentLabel.empty())
+                {
+                    info << Execution::ManifestInfoEmphasis << documentation.DocumentLabel << ": "_liv;
+                }
+
+                if (!documentation.DocumentUrl.empty())

```

Comment:
Changed the logic a bit so that it checks for the URL first as I would argue that the `documentUrl` is the more "important" piece of information. If no label exists, it will just display the documentationUrl.